packages/bot-web-ui/src/components/dashboard/dashboard-component/cards.tsx

```diff
@@ -35,6 +37,32 @@ const Cards = observer(({ is_mobile, has_dashboard_strategies }: TCardProps) =>
     const { handleFileChange, loadFileFromLocal } = load_modal;
     const { setFormVisibility } = quick_strategy;
 
+    //this is to check on click of which icon the user has come to bot builder
+    const sentToRudderStackTabChange = (type: string) => {
+        Analytics.trackEvent('ce_bot_builder_form', {
+            shortcut_name: type,
+            form_source: 'bot_dashboard_form-shortcut',
```

Comment:
does this proper source for that action?

Reply:
let's remove form_source here then

Reply:
yes i have already added action there 
![Uploading image.png…]()